src/js/components/NavTabsComponent.jsx

```diff
@@ -1,21 +1,38 @@
 var classNames = require("classnames");
 var React = require("react/addons");
 
+var DeploymentEvents = require("../events/DeploymentEvents");
+var DeploymentStore = require("../stores/DeploymentStore");
+var DeploymentActions = require("../actions/DeploymentActions");
+
 var NavTabsComponent = React.createClass({
   displayName: "NavTabsComponent",
 
   propTypes: {
     activeTabId: React.PropTypes.string.isRequired,
+    activeDeployments: React.PropTypes.number,
```

Comment:
Yes, that should be removed here. We don't need to test so called 'activeDeployments', we already test for the length of the deployments array in the store.